packages/slice-machine/src/tracker.ts

```diff
@@ -63,34 +57,41 @@ export class SMTracker {
       );
   }
 
-  async #identify(userId: string): Promise<void> {
+  async #page(attributes: Record<string, unknown> = {}): Promise<void> {
     if (!this.#isTrackingPossible(this.#client)) {
       return;
     }
 
     return this.#client
       .then((client): void => {
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        client.identify(userId);
+        client.page(attributes);
       })
-      .catch(() => console.warn(`Couldn't report identify: Tracking error`));
+      .catch(() => console.warn(`Couldn't report page event: Tracking error`));
   }
 
-  async #group(attributes: Record<string, unknown> = {}): Promise<void> {
+  async #identify(userId: string): Promise<void> {
     if (!this.#isTrackingPossible(this.#client)) {
       return;
     }
 
-    const repoName = this.#repoName;
+    return this.#client
+      .then((client): void => {
```

Comment:
why void?
plus if you add void before the `client.identify` promise the warning should go away

Reply:
Because this function return nothing, just calling the segment segment.

Reply:
I add the void on the promise to remove the eslint comment